docs/appendix/upgrade-notes/3.x-to-4.0.rst

```diff
@@ -49,6 +49,31 @@ Miscellaneous changes
 * The config value for ``servicehandler`` has been removed
 * In certain edge cases the default value of an API parameter will not be applied
 
+Plugin Manifest
+---------------
+
+The plugin manifest file is no longer used. Features of the manifest have been removed or moved to different locations.
+
+It is no longer possible to require a specific php ini setting.
+
+php version requirement -> composer require
+php extension requirement -> composer require
+plugin conflicts -> composer conflicts
+plugin requirement -> elgg-plugin
+plugin position requirement -> elgg-plugin
+
+plugin version -> elgg-plugin
+plugin activate on install -> elgg-plugin
+plugin name -> activate on install
```

Comment:
plugin name -> ???

Reply:
uh... thanks for reviewing :)

Reply:
I've missed it before but the plugin name line was already there before. So, it's listed twice now.

As far as I know the "suggests" tag was already deprecated in Elgg 3.3. May I suggest to include the usage of "suggest" in composer.json in any docs (if any documentation about usage of composer.json exists / is planned to be added).